### DISCUSSION TITLE
Use inline css inside an HTML section that was "replaced" by a template value

### DISCUSSION THREAD

**Discussion by ghita** [handlebars-lang/handlebars.js#1906] [Q&A]
Hi all,

I use handlebars like this somewhere inside the body of the HTML
`<body>
<div>
   {{template.header}}
<div/>
<body/>`

Everything works as expected till I want to add inline css styles inside the "header" content
If I try to use <elem style=""> inside I get them in the outputted result, probably as  expected because I don't specify somehow that the template owner wants control over the styling inside that template "value"

Is there a way for the user to specify styling for outside and handlebars just output the header with that ?

**Comment by jaylinski** [comment]
Hi! I'm not sure I understand your problem. Do the CSS styles get rendered? Maybe you need to use raw-output via `{{{ template.header }}}`?

**Comment by ghita** [comment]
@jaylinski Yes, the solution was to use raw-output via {{{}}}. Thanks for confirmation !
